modules/app-connector/src/test/test.ts

```diff
@@ -66,17 +67,29 @@ describe("should work", () => {
           assert(JSON.stringify(result).indexOf("error") <= -1);
         });
     });
-    it("send event", () => {
+    it("send legacy event", () => {
       let eventData = {
         "event-type": "customer.created",
         "event-type-version": "v1", //event types normally end with .v1
         "event-time": new Date().toISOString(),
         data: JSON.parse(eventPublishAPI),
       };
-      return event.send(eventData).then((result: any) => {
+      return event.sendLegacyEvent(eventData).then((result: any) => {
         expect(JSON.stringify(result)).to.match(new RegExp(JSON.stringify(JSON.parse(eventResponseExpected)), "g"));
       });
     });
+    it("send cloud event", () => {

```

Comment:
Here we're sending a structured CloudEvent, could we also send a CloudEvent in binary mode?